integration-tests/api/__tests__/admin/fulfillment-sets.spec.ts

```diff
@@ -120,6 +120,77 @@ medusaIntegrationTestRunner({
             ]),
           })
         )
+
+        const serviceZoneId = fset.service_zones[0].id
+
+        const countryGeoZone = fset.service_zones[0].geo_zones.find(
+          (z) => z.type === "country"
+        )
+
+        // Updates an existing and creates a new one
+        const updateResponse = await api.post(
+          `/admin/fulfillment-sets/${fulfillmentSetId}/service-zones/${serviceZoneId}`,
+          {
+            name: "Test Zone Updated",
+            geo_zones: [
+              {
+                id: countryGeoZone.id,
+                country_code: "us",
+                type: "country",
+              },
+              {
+                country_code: "ca",
+                type: "country",
+              },
+            ],
+          },
+          adminHeaders
+        )
+
+        const updatedFset = updateResponse.data.fulfillment_set
+
+        expect(updateResponse.status).toEqual(200)
+        expect(updatedFset).toEqual(
+          expect.objectContaining({
+            name: "Fulfillment Set",
+            type: "shipping",
+            service_zones: expect.arrayContaining([
+              expect.objectContaining({
+                id: serviceZoneId,
+                name: "Test Zone Updated",
+                fulfillment_set_id: updatedFset.id,
+                geo_zones: expect.arrayContaining([
+                  expect.objectContaining({
+                    id: countryGeoZone.id,
+                    country_code: "us",
+                    type: "country",
+                  }),
+                  expect.objectContaining({
+                    country_code: "ca",
+                    type: "country",
+                  }),
+                ]),
+              }),
+            ]),
+          })
+        )
+
+        const deleteResponse = await api.delete(
+          `/admin/fulfillment-sets/${fulfillmentSetId}/service-zones/${serviceZoneId}`,
+          adminHeaders
+        )
+
+        expect(deleteResponse.status).toEqual(200)
+        expect(deleteResponse.data).toEqual(
+          expect.objectContaining({
+            id: serviceZoneId,
+            object: "service-zone",
+            deleted: true,
+            parent: expect.objectContaining({
+              id: fulfillmentSetId,
```

Comment:
q: is that a new convention?

Reply:
Yes, when deleting children, we include the parent